### DISCUSSION TITLE
Cloudflare D1 support

### DISCUSSION THREAD

**Discussion by robingenz** [beekeeper-studio/beekeeper-studio#2093] [Q&A]
First of all, thanks for this great project!
I am currently working with Cloudflare D1 and would like to use Beekeeper Studio as SQL editor. Is there any way to do this similar to what [TablePlus](https://twitter.com/TablePlus/status/1776611556509253687) offers?

**Comment by zuflucht** [comment]
I am also looking for such a feature. 
 anything new?

**Comment by kholisrag** [comment]
👍🏽 for this features
